package.json

```diff
@@ -26,8 +26,10 @@
   },
   "dependencies": {
     "@types/jest": "^24.0.18",
+    "axios": "^0.19.0",
     "express": "^4.17.1",
     "faker": "^4.1.0",
+    "fs": "0.0.1-security",
```

Comment:
Curious as to why this was added as a dependency? Maybe added as part of another package?

Reply:
I eventually intend to use it to obtain photos from a better random source, to then upload those photos to another source (like AWS). Stretch goal.